teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(age, specialization, parent_permission, name = 'unknown')
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure the `parent_permission` parameter also has `true` as the default value. You only updated the `name` parameter and missed the `parent_permission`. Make sure you also fix that. 
Important, you should define `parent_permission` as `boolean` and not a `string` (.i.e `parent_permission: true` and not `parent_permission: 'true'`)